manager/container.go

```diff
@@ -356,7 +372,7 @@ func (self *containerData) nextHousekeeping(lastHousekeeping time.Time) time.Tim
 		}
 	}
 
-	return lastHousekeeping.Add(self.housekeepingInterval)
+	return lastHousekeeping.Add(jitter(self.housekeepingInterval, JitterFactor))
```

Comment:
Why? If we're jittering in housekeeping then this seems to be enough to spread load across the housekeeping period.